docs/spec.md

```diff
@@ -162,6 +162,12 @@ type Spec = ArraySpec | BooleanSpec | NumberSpec | ObjectSpec | StringSpec;
 | readAsMethod | `'readAsArrayBuffer'` `'readAsBinaryString'` `'readAsDataURL'` `'readAsText'` |          | File reading method                                                                    |
 | ignoreText   | `boolean`                                                                     |          | For `true`, will show the `File uploaded` stub instead of the field value              |
 
+#### DateInput
+
+| Property     | Type     | Required | Description                                                                             |
+| :----------- | :------- | :------: | :-------------------------------------------------------------------------------------- |
+| outputFormat | `string` |          | Format returning string. [Available formats](https://day.js.org/docs/en/display/format) |
+
```

Comment:
I think there are two formats needed here, one format that is waiting for the backend, the second format is the one in which we will display the date to the user

It is worth considering that the backend can wait for a millisecond string or a timestamp object, I did not see it here https://day.js.org/docs/en/display/format